idrstream/cellprofiler_metadata.py

```diff
@@ -0,0 +1,20 @@
+"""
+This function will convert the `data_to_process.tsv` file into a .csv so that CellProfiler can read in the metadata and connect the appropriate
+metadata to the images from each batch. 
+"""
+import pandas as pd
+
+def convert_tsv_to_csv(data_to_process_tsv: str):
+    """ Convert `data_to_process.tsv` file into a .csv for CellProfiler to read in when processing the images
+
+    Parameters
+    ----------
+        data_to_process_tsv : str
+            string path to the `data_to_process.tsv` file
+    """
+    # read in metadata tsv file
+    data_to_process_csv = pd.read_table(data_to_process_tsv,sep='\t')
+    
+    # converting tsv file into csv
+    data_to_process_csv.to_csv('example_files/data_to_process.csv',index=False)
```

Comment:
Does this mean that CellProfiler can read the data in the same format that I decided on for `IDR_stream` input, it just can't load a csv?

Reply:
I am not sure what you mean here. It can not load in a `tsv` which is the input format for `IDR_stream` for the data_to_process file. That is why it must be converted to a `csv` so it can load it in.

Reply:
I just meant that you simply load the tsv and save the same dataframe as csv without modifying the dataframe at all. I would be surprised if CP can use the data from the dataframe as it looks for `IDR_stream`. But maybe you modify it somewhere else? We can also discuss Friday as this function is still confusing me.

Reply:
Also make sure to change `example_files/data_to_process.csv` so we aren't using absolute pathing!